test_unstructured/chunking/test_basic.py

```diff
@@ -0,0 +1,108 @@
+"""Unit-test suite for the `unstructured.chunking.basic` module.
+
+That module implements the baseline chunking strategy. The baseline strategy has all behaviors
+shared by all chunking strategies and no extra rules like perserve section or page boundaries.
+"""
+
+from __future__ import annotations
```

Comment:
This line enables the so-called _postponed annotations_ behavior.  Postponed annotations removes the need to quote forward references in most cases and provides some other typing behavior improvements. This is described in [PEP 563](https://peps.python.org/pep-0563/).

Postponed annotations was originally going to be the default in Python 3.10 but was postponed. I can't find a hard commit by the Python team on "postponed to when", but I expect we can assume it would be at least Python 3.14 since the "New in Python 3.13" page makes no mention of it.